components/brandprofile/TypographyShowcase.tsx

```
@@ -66,8 +302,54 @@ const TypographyShowcase = () => {
+        </p>
+        <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4 mb-8">
+          {textColors.map(color => (
+            <button
+              key={color.name}
+              onClick={() => setSelectedTextColor(color.combinedClass)}
+              className={`p-4 rounded-lg border-2 transition-all duration-200 text-left hover:shadow-md ${
+                selectedTextColor === color.combinedClass
+                  ? 'border-primary-500 bg-primary-50 dark:bg-primary-900/20'
+                  : 'border-secondary-200 dark:border-secondary-700 hover:border-secondary-300 dark:hover:border-secondary-600'
+              }`}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)